### PR TITLE
[PPML]Update prerequisite with enclave-key file

### PR DESCRIPTION
**Pull request by ManfeiBai**

### CODE REVIEW

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md [hunk 1]
```diff
@@ -47,6 +48,12 @@ Check SGX driver with `ls /dev | grep sgx`. If SGX driver is not installed, plea
 ./ppml/scripts/install-graphene-driver.sh
 ```
 
+(2) You also need to generate your enclave key using the command below, and keep it safely for future remote attestations and to start SGX enclaves more securely. It will generate a file enclave-key.pem in your present working directory, which will be your enclave key. To store the key elsewhere, modify the outputted file path.
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/ppml.md:51]: (2) Generate key for enclave

Comment (ManfeiBai) [reply docs/readthedocs/source/doc/PPML/Overview/ppml.md]: Done, thanks~

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md [hunk 2]
```diff
@@ -37,7 +37,8 @@ In this section, we will demonstrate how to use Analytics-Zoo to setup trusted S
 
 ### Get started
 
-#### Prerequisite: Install SGX Driver & Prepare Scripts
+#### Prerequisite: 
+(1)Install SGX Driver & Prepare Scripts
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/ppml.md:41]: Add a space after (1)

Comment (ManfeiBai) [reply docs/readthedocs/source/doc/PPML/Overview/ppml.md]: Done, thanks~

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**